addons/knobs/src/components/Panel.js

```diff
@@ -146,14 +130,22 @@ export default class Panel extends React.Component {
       const knobKeyGroupId = knobs[key].groupId;
       groupIds.push(knobKeyGroupId);
       groups[knobKeyGroupId] = {
-        render: () => <div id={knobKeyGroupId}>{knobKeyGroupId}</div>,
+        render: ({ active: groupActive, selected }) => (
+          <TabWrapper active={groupActive || selected === DEFAULT_GROUP_ID}>
+            <PropForm
+              knobs={knobsArray.filter(knob => knob.groupId === knobKeyGroupId)}
+              onFieldChange={this.handleChange}
+              onFieldClick={this.handleClick}
+            />
+          </TabWrapper>
+        ),
         title: knobKeyGroupId,
       };
     });
 
-    if (groupIds.length > 0) {
+    if (groupIds.length >= 0) {
```

Comment:
what happens in the `== 0` case?

Reply:
Sorry, left a bit of a mess here, the length of an array is always `>= 0`..

What happens is: